### PR TITLE
Fix error on homepage load

### PR DESCRIPTION
**Pull request by corymcdonald**
#### Features

Because we weren't tracking Uphold IDs on the card connections we ran into an issue where if someone tries and create new cards then there will be an error.

### CODE REVIEW

File: app/jobs/create_uphold_channel_card_job.rb [hunk 1]
```diff
@@ -57,12 +60,15 @@ def find_or_create_card(uphold_connection, channel)
     [card_id, upfc]
   end
 
+  def card_exists?(uphold_connection, card_id)
+    return if card_id.blank?
```
Comment (yachtcaptain23) [inline app/jobs/create_uphold_channel_card_job.rb]: `return false` is preferable

File: app/jobs/create_uphold_channel_card_job.rb [hunk 2]
```diff
@@ -57,12 +60,15 @@ def find_or_create_card(uphold_connection, channel)
...
+
+    uphold_connection.uphold_client.card.find(uphold_connection: uphold_connection, id: card_id).present?
+  rescue Faraday::ResourceNotFound
```
Comment (yachtcaptain23) [inline app/jobs/create_uphold_channel_card_job.rb]: I think it should return `false`

File: app/jobs/create_uphold_channel_card_job.rb (hunk 1 above)
Comment (corymcdonald) [reply app/jobs/create_uphold_channel_card_job.rb]: https://github.com/brave-intl/publishers/pull/2200/commits/80a3639f7c23cf850d130406ef3db91bc8eb733d

File: app/jobs/create_uphold_channel_card_job.rb (hunk 2 above)
Comment (corymcdonald) [reply app/jobs/create_uphold_channel_card_job.rb]: https://github.com/brave-intl/publishers/pull/2200/commits/80a3639f7c23cf850d130406ef3db91bc8eb733d

**Pull request merged**